pyproject.toml

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[tool.poetry]
+name = "sdap-nexus"
+version = "1.2.0"
+description = "Nexus component for Apache SDAP"
+authors = ["SDAP PPMC <dev@sdap.apache.org>"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [
+    { include = "webservice", from = "analysis" },
+    { include = "nexustiles", from = "data-access" }
+]
+repository = "https://github.com/apache/incubator-sdap-nexus"
+exclude = ["docs", "tests"]
+classifiers=[
+    'Development Status :: 6 - Mature',
+    'Intended Audience :: Science/Research',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3.8',
+    "License :: OSI Approved :: Apache Software License"
+]
+
+
+[tool.poetry.dependencies]
+python = "~=3.8"
+netcdf4 = "1.5.5.1"
```

Comment:
I basically translated the existing requirements.txt & conda-requirements.txt files to pyproject.toml, adjusting as needed. They also pin to exact versions and I was trying to get the conda -> poetry switch done quickly so wanted to avoid having to mess around with dependencies when we had a set that were proven to work